lib/modules/hosts/iloopit.js

```diff
@@ -3,32 +3,26 @@ addLibrary('mediaHosts', 'iloopit', {
 	domains: ['iloopit.net'],
 	logo: '//iloopit.net/favicon.ico',
 	detect: href => (
-		(/^https?:\/\/(?:\w+\.)?iloopit\.net\/tube\/(\d+)/i).exec(href) ||
-		(/^https?:\/\/(?:\w+\.)?iloopit\.net\/[\-a-z\ \/%20]+?\/\?type=looplayer&loopid=(\d+)/i).exec(href) ||
-		(/^https?:\/\/(?:\w+\.)?iloopit\.net\/(\d+)\/[a-z\-\/]+((\?type=looplayer)|(\?type=embed))?/i).exec(href)
+		(/^https?:\/\/(?:\w+\.)?iloopit\.net\/.+?\/\?type=looplayer&loopid=(\d+)/i).exec(href) ||
+		(/^https?:\/\/(?:\w+\.)?iloopit\.net(?:\/tube\/)?\/(\d+)\/.+?\/(?:(?:\?type=looplayer)|(?:\?type=embed))?/i).exec(href)
 	),
 	async handleLink(elem, [, hash]) {
-		const info = await RESEnvironment.ajax({
-			url: `https://iloopit.net/api/video/${hash}`,
-			type: 'json'
-		});
+		let link = '';
+		const testWithTitle = /iloopit\.net(?:\/tube\/)?\/(\d+)\/(.+)?\//;
+		const titleResult = testWithTitle.exec(elem.href);
 
-		elem.type = 'VIDEO';
+		if (titleResult) {
+			link = `https://iloopit.net/${titleResult[1]}/${titleResult[2]}/?type=embed`;
+		} else {
+			link = elem.href.replace('type=looplayer', 'type=embed');
+		}
 
 		elem.expandoOptions = {
```

Comment:
Neither the `autoplay` nor `loop` options are used by iframe expandos, so `elem.expandoOptions` should be removed.